<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/stick_vlp32imu4cam/with_vicon/20201213_183636.bag"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find msg_conversion)/rviz_config/posestamped_reset_origin.rviz"/>

	<!-- MAIN -->
	<node pkg="msg_conversion" type="tf_to_posestamped" name="tf_to_posestamped" output="screen">
		<param name="parent_frame_id" type="string" value="/world"/>
		<param name="child_frame_id" type="string" value="/vicon/sensors/sensors"/>
	</node>
	<node pkg="msg_conversion" type="posestamped_reset_origin" name="posestamped_reset_origin" output="screen">
		<remap from="/pose" to="/tf_to_posestamped"/>
		<param name="parent_frame_id" type="string" value="/world"/>
		<param name="child_frame_id" type="string" value="/tf_to_posestamped/reset"/>
	</node>
</launch>
